### PR TITLE
Adding updateCollection to jets

### PR DESCRIPTION
**Pull request by alefisico**
- Using updateCollection from jetToolbox to use jet collections from miniAOD (slimmedJets, slimmedJetsPuppi, slimmedJetsAK8) and avoid reclustering.
- Although this PR improves a lot the framework, it needs some more work.
- This version is good for test

### CODE REVIEW

**Review by dmajumder — CHANGES_REQUESTED** [review]
@alefisico OK, so with the 'updateJetCollection' what are we adding to the jets?
- is it possible to add (unless there) the PUPPI jet substructure infos?
- B tagging should be rerun since we have a new training for the double b tagger, some bug fixes for the c tagger, and looser track requirements for b tagging in general.

**Comment by alefisico** [comment]
so what exactly do you want to add? In this PR only the info that was not miniAOD is added, like trimmed/filtered jet mass. But of course I can add whatever we extra we want. Can you please give me a list of things that you want to include

File: test/b2gedmntuples_cfg.py [hunk 1]
```diff
@@ -402,6 +393,25 @@
 jetAlgoAK8      = 'AK8PFchs'
 jetAlgoAK8Puppi = 'AK8PFPuppi'
 
+ak4Cut='pt > 25 && abs(eta) < 5.'
+ak8Cut='pt > 200 && abs(eta) < 2.4'
+jetToolbox( process, 'ak4', 'analysisPath', 'edmNtuplesOut', runOnMC=runMC, updateCollection=jetAK4Label, JETCorrPayload=jetAlgo, JETCorrLevels=corrections, addQGTagger=True ) 
+jetToolbox( process, 'ak4', 'analysisPath', 'edmNtuplesOut', runOnMC=runMC, updateCollection=jetAK4LabelPuppi, JETCorrPayload='AK4PFPuppi', JETCorrLevels=['L2Relative', 'L3Absolute'] )  
+jetToolbox( process, 'ak8', 'analysisPath', 'edmNtuplesOut', runOnMC=runMC, updateCollection=jetAK8Label, JETCorrPayload=jetAlgoAK8, addTrimming=True, rFiltTrim=0.1, addFiltering=True ) #, addSoftDropSubjets=True, addPruning=True, addSoftDrop=True, addCMSTopTagger=True , addNsubSubjets=True, subjetMaxTau=4 ) #, addNsub=True, bTagInfos=listBTagInfos, bTagDiscriminators=listBtagDiscriminatorsAK8, Cut=ak8Cut
```
Comment (dmajumder) [inline test/b2gedmntuples_cfg.py:399]: @alefisico  would like to rerun the b tagging for AK8PFCHS also

**Comment by alefisico** [comment]
@dmajumder I was about to included what you suggest, should I do it or not?

**Comment by dmajumder** [comment]
@alefisico yes, you can create another PR for this.

**Pull request merged**